vector/src/main/java/im/vector/app/features/command/CommandParser.kt

```diff
@@ -61,6 +62,20 @@ object CommandParser {
                 return ParsedCommand.ErrorEmptySlashCommand
             }
 
+            // If the command is not supported by threads return error
+
+            if (BuildConfig.THREADING_ENABLED && isInThreadTimeline) {
+                val slashCommand = messageParts.first()
+                val notSupportedCommandsInThreads = Command.values().filter {
```

Comment:
Maybe move this val as a lazy class member to avoid computing it each time a message is sent in a thread

Reply:
keep in mind `CommandParser` is a **static** class (object). Can you explain a bit more what you mean?

Reply:
Now it's clear! Totally will do that +1

Reply:
Done!